modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/meetings/portlet/MeetingsPortlet.java

```diff
@@ -439,6 +443,55 @@ public void updatePowwowMeeting(
 		writeJSON(actionRequest, actionResponse, jsonObject);
 	}
 
+	public void updatePowwowMeetingOccurrence(
+		ActionRequest actionRequest, ActionResponse actionResponse)
+			throws Exception {
+
+		ServiceContext serviceContext = ServiceContextFactory.getInstance(
+			PowwowMeeting.class.getName(), actionRequest);
+		ThemeDisplay themeDisplay = (ThemeDisplay)actionRequest.getAttribute(
+			WebKeys.THEME_DISPLAY);
+
+		long powwowMeetingId = ParamUtil.getLong(actionRequest, "powwowMeetingId");
+		long occurrenceId = ParamUtil.getLong(actionRequest, "occurrenceId");
+
+		try {
+			PowwowMeetingOccurrence powwowMeetingOccurrence =
+				PowwowMeetingOccurrenceLocalServiceUtil
+					.fetchPowwowMeetingOccurrence(occurrenceId);
+
```

Comment:
should have a validation check: new start time not before now. and maybe ensure end time not before start time, if the UI components did not handle that